helpers/helpers.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2023 IndyKite
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+
+	ingestpb "github.com/indykite/indykite-sdk-go/gen/indykite/ingest/v1beta2"
+)
+
+// DeleteNodesWithTypeNode is a helper function that delete all nodes of specific type either DigitalTwin
+// or Resource and return array of StreamRecordsResponse.
+func (c *Client) DeleteNodesWithTypeNode(
```

Comment:
Why `DeleteNodesWithTypeNode` and not just `DeleteNodesWithType`? I don't understand the `Node` at the end